.github/workflows/ESlint.yaml

```
@@ -0,0 +1,47 @@
+name: ESlint
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
```

```suggestion
on:
  push:
    branches:
      - master
      - main
    pull_request:
```

Não vejo necessidade de limitar a execução do lint de tal maneira, o que acha de alterar para executar em todos os `pull_request` e em push na branch default?

Deixei `main` e `master` pois logo renomearemos a branch.